openapi/platform/platform.yaml

```diff
@@ -1135,19 +1134,19 @@ components:
 
         All amounts will be specified in the minor unit of the invoice's currency. For example in the case of USD that would be cents. For more info on converting between minor and major units see [this wikipedia article](https://en.wikipedia.org/wiki/ISO_4217#Minor_units_of_currency).
       x-tags:
-        - wholesaleInvoices
+        - payableInvoices
       properties:
         id:
           type: string
           description: 'A platform generated identifier for the invoice '
         type:
           type: string
           enum:
-            - wholesaleInvoices
+            - payableInvoices
         attributes:
           type: object
           properties:
-            createdAt:
+            created:
```

Comment:
Please keep this as `createdAt`. We want to establish a convention of strings that are formatted as `date-time` ending with `At`.